automation/taskcluster/lib/tasks.py

```diff
@@ -389,7 +389,7 @@ def craft_nightly_signing_task(self, build_task_id, apk_paths, is_staging=True):
     def _craft_signing_task(
         self, name, description, signing_type, assemble_task_id, apk_paths, routes, treeherder
     ):
-        signing_format = "autograph_apk"
+        signing_format = 'autograph_apk_reference_browser'
```

Comment:
Regressed at https://github.com/mozilla-mobile/reference-browser/commit/4634f981a1e0dc14c3ded855efdd7664ef6baf74#diff-59f90d509bc6ae2c9d877250c5ba49c5L349

Reply:
:thinking: weird, I remember writing "autograph_apk", but I'm not sure why.
Good catch

Reply:
I shamelessly copied what's on Fenix. I'm not surprised the formats differ as they run on the same worker.